app/src/main/res/layout-sw600dp/pregamepage.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.percent.PercentRelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/autoPageLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+
+        <TableLayout
+            android:id="@+id/pregameTableLayout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentTop="true">
+
+            <TableRow
+                android:id="@+id/startPosition"
+                android:layout_width="fill_parent"
+                android:layout_marginBottom="17dp"
+                android:weightSum="4">
+
+                <TextView
+                    android:id="@+id/pregame0"
```

Comment:
It's probably a good idea to get this id changed to something more descriptive at some point, though it does not matter much as it's just a textview.

Reply:
I might just remove it outright

Reply:
It probably was just there because of us using a relative layout at some point.